### PR TITLE
fixes #848

### LINKED ISSUE #848: incorrect identifiers in Swarm "4.6.7. Interacting with the Secured Booster"
This chapter uses `secured-wf-http` as an identifier of the route/service, but that's wrong. The correct one is `wfswarm-rest-http-secured`.

### PR DESCRIPTION
**Pull request by rhoads-zach**
@tradej @metacosm @Ladicek let me know what you think

### CODE REVIEW

File: docs/topics/proc_getting-the-secured-booster-api-endpoint.adoc [hunk 1]
```diff
@@ -33,10 +33,10 @@ A sample output is shown in the following table:
 | <all>
 | passthrough
 
-| secured-{parameter-runtime}-http
-| secured-{parameter-runtime}-http-myproject.LOCAL_OPENSHIFT_HOSTNAME
+| {parameter-runtime}-rest-http-secured
```
Comment (metacosm) [inline docs/topics/proc_getting-the-secured-booster-api-endpoint.adoc]: if `{parameter-runtime}` is `spring-boot` for the Spring Boot runtime, then yes. Hmm, on second thought, maybe not. This value is actually the artifact Id of the Maven project. While this value is correct for the Spring Boot runtime  booster *before* Launcher takes it and "clones" it with the user having the possibility to change the artifact id. So in fact, we don't know what that value may be until runtime…

Comment (rhoads-zach) [reply docs/topics/proc_getting-the-secured-booster-api-endpoint.adoc]: in this case, the parameter is "sb".

what if we changed the whole thing to <NAME_OF_ARTIFACT> or something similar?

Comment (metacosm) [reply docs/topics/proc_getting-the-secured-booster-api-endpoint.adoc]: `sb` is not correct, then. But in any case, this needs to be changed. The problem with using <NAME_OF_ARTIFACT> is that we don't currently have no explanation of the launch process to reference and explain that the artifact id is what was chosen when the booster was created in Launch (or the default value generated by Launch, which is *not* the value that we set in the boosters)… This needs to be well documented and it is not trivial to do so.

**Comment by rhoads-zach** [comment]
@metacosm updated with generic value and explanation. Let me know what you think.

**Comment by metacosm** [comment]
LGTM

**Comment by rhoads-zach** [comment]
thanks @metacosm, merging.

File: docs/topics/proc_getting-the-secured-booster-api-endpoint.adoc [hunk 2]
```diff
@@ -50,3 +50,4 @@ A sample output is shown in the following table:
 ====
 --
 
+IMPORTANT: `PROJECT_ID` is based on the name you entered when generating your booster using link:{link-launcher-oso}[{launcher-oso}] or the {launcher} tool. The default value for `PROJECT_ID` is `{parameter-runtime}-rest-http-secured`.
```
Comment (Ladicek) [inline docs/topics/proc_getting-the-secured-booster-api-endpoint.adoc]: The concept of "default value" IMHO doesn't make a lot of sense to me here, and the value isn't actually correct :-) See https://github.com/openshiftio-vertx-boosters/vertx-secured-http-booster/blob/master/pom.xml#L29

**Pull request merged**